app/uploaders/base_uploader.rb

```diff
@@ -0,0 +1,32 @@
+class BaseUploader < CarrierWave::Uploader::Base
+  include CarrierWave::Compatibility::Paperclip
+  include CarrierWave::MiniMagick
+  include CarrierWave::Backgrounder::Delay
+
+  storage :fog
+
+  # Store original size
+  version :original
+
+  # Process cropping on upload
+  process :cropping
+
+  def store_dir
```

Comment:
Hm, this method is already defined by Carrierwave's Paperclip compatibility module ...  https://github.com/carrierwaveuploader/carrierwave/blob/d41ad71ad71a813dddf47e750e5a8b5f5c8d4e0d/lib/carrierwave/compatibility/paperclip.rb#L74

Any reason why this is required here?

Reply:
It's getting overriden to match the format with what we had before